src/main/scala/edu/ie3/osmogrid/exception/UnsupportedRequestException.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.osmogrid.exception
+
+case class UnsupportedRequestException(
```

Comment:
dead code ☠️ 

(`RunSupport:74` uses `scala.annotation.migration.UnsupportedOperationException` which is a type pointing to `java.lang.UnsupportedOperationException`)

I would recommend a) renaming this Exception to avoid further confusion or b) remove this class

Reply:
Just to be clear: This is `UnsuuortedRequestException`, whereas I accidentally used `UnsupportedOperationException` in `RunSupport`. So, naming issue resolved?